packages/graphql-testing/src/Requests.ts

```diff
@@ -0,0 +1,51 @@
+import {Operation} from 'apollo-link';
```

Comment:
copy from https://github.com/Shopify/quilt/tree/master/packages/jest-mock-apollo
than added the constructor